UnitTests/MockedEventsFixture.cs

```diff
@@ -352,7 +352,7 @@ public void DoesNotRaiseEventOnSubObject()
 
 			mock.Raise(p => p.Adder.Added += null, EventArgs.Empty);
 
-			Assert.False(raised);
+			Assert.True(raised);
```

Comment:
Note that this is in effect a functional change. The changes of this breaking client code are probably quite small (see https://github.com/Moq/moq4/issues/166#issuecomment-311574982).

At the same time, replacing this unit test with its exact opposite also has the advantage of fixing an inconsistency, since the quickstart guide already documents the ability to raise events on sub-objects.